app/views/common_parameters/_parent_parameter.html.erb

```diff
@@ -7,7 +7,7 @@
           <%= content_tag(:input, '', :class => "form-control", :disabled => true,  :value => _('Global'), :type=> :text ) %>
         </td>
         <td class="col-md-2">
-          <%= text_field_tag("parent_parameter_#{i}_key", '', :class => "form-control", :placeholder => parameter.name) %>
+          <%= text_field_tag("parent_parameter_#{i}_key", '', :class => "form-control", :value => parameter.name) %>
```

Comment:
@domcleal I agree, I'll change it.
This makes me think about removing the parameter. If you can't send an empty value or change the key to something else there is no way to remove a parameter that's coming from a parent so maybe there should be a remove button?